docs/source/upgrade-guide/index.md

```diff
@@ -522,6 +522,63 @@ The `react/jsx-key` rule has been enabled in ESlint for catching missing `key` i
 You might catch some violations in your project or add-on code after running ESlint.
 Adding the missing `key` property whenever the violation is reported will fix it.
 
+### `@plone/registry` moved to ESM
+
+The `@plone/registry` package has been moved to ESM.
+The Add-on Registry scripts have also been refactored to TypeScript.
+For maximum compatibility with CommonJS builds, the default exports have been moved to named exports.
```

Comment:
I've already evaluated the possibility, but it has a downside: It does not allow other named exports to exist along with it:

```
One inconvenience to this approach is that you’re not allowed to include other top-level named exports alongside the export =:
```

I thought that it might be better not to mess around with them and move all to be named. Also the fact of having a default and some named does not have much value anyways, also the usage becomes easier to remember afterwards.

The user does not really have to mess around with it (except for the files mentioned in the upgrade guide), and all is kept internally. For the out-of-Volto experience, I think that the solution is better too and unifies things.

Reply:
@sneridagh Ok, right. I thought that didn't apply to these since they use the default export, but I was just wrong: of course they have named exports too.